src/applications/vaos/sass/vaos.scss

```diff
@@ -95,11 +95,19 @@
   width: 20%;
 }
 
+.vaos-calendar__calendar-week {
+  flex-wrap: wrap;
+}
+
+.vaos-calendar__flex-line-break {
+  flex-basis: 100%;
+  height: 0;
+}
 
 .vaos-calendar__calendar-day {
-  width: 18%;
+  flex-grow: 1;
+  flex-basis: 0;
```

Comment:
You might be able to apply `vads-u-flex--1` and get the same thing.

Reply:
I've addressed one or two of the comments, but this is what I was referring to the other day about some judgment calls around using these utility classes.

I think all of the remaining comments are comments about specific properties within classes that have various other properties. Some of those properties can be covered by design system utilities, some cannot. When they can all be covered, in this calendar widget in particular, it would result in elements with maybe 5 or six classes, which makes the markup pretty awkward to read and edit.

Additionally, you're asking us to use utilities for some (but not all) of the properties in a given class, which means splitting styles between a class in this css file and utility classes in the markup for individual elements. This makes for an awkward reading and editing experience, since I can't expect styles for a single element to be defined in one place.

I get the reasoning behind the utility classes and helped out with the implementation of some of them. They're hugely useful overall. But I think for widgets and components that require a lot of styling work, their overuse can introduce some maintainability issues. This is a judgment call though and I'm curious if you all have thought about the best way to approach and address some of this. "Use the utitlity classes" is a good general rule, but I think there are cases where there's more nuance involved.

It's also worth noting that there are some other issues in this file that we need to fix, like the unnecessary nesting and some media query styles that aren't written mobile-first, which have been more painful issues on va.gov over the past couple years.

Reply:
Yeah, I was a bit hesitant about leaving some of these considering that a lot of styling already existed in custom classes and splitting it between that and `vads` classes would add some confusion.

"Use the utility classes" does seem like a good general rule, but I think they're better used sparingly otherwise we run into the situation you mention where elements have 5+ classes and the code gets ugly.  Perhaps we'll be able to come up with something that is consistent, flexible, _and_ easy to use :crossed_fingers: